contributors/guide/README.md

```diff
@@ -73,12 +73,16 @@ You get the idea - if you ever see something you think should be fixed, you shou
 
 There are multiple repositories within the Kubernetes community and a full list of repositories can be found [here](https://github.com/kubernetes/).
 Each repository in the Kubernetes organization has beginner-friendly issues that provide a good first issue. For example, [kubernetes/kubernetes](https://git.k8s.io/kubernetes) has [help wanted](https://go.k8s.io/help-wanted) and [good first issue](https://github.com/kubernetes/kubernetes/labels/good%20first%20issue) labels for issues that should not need deep knowledge of the system.
```

Comment:
not really related to this PR: but are the help-wanted and good-first-issue labels only for k/k?

Reply:
Nope. All repos :)